src/commands/build/wranglerjs/mod.rs

```diff
@@ -65,6 +66,8 @@ pub fn run_build(
             .to_string();
         command.arg("--no-webpack-config=1");
         command.arg(format!("--use-entry={}", package_main));
+    } else {
+        command.arg(format!("--webpack-config={}", &webpack_config_path.to_str().unwrap().to_string()));
```

Comment:
```suggestion
        command.arg(format!("--webpack-config={}", &webpack_config_path.to_str()));
```
Would that work? I'm also not a Rust expert

Reply:
It does not

Reply:
try this. (? operator will return Err() if there is a problem instead of doing a panic like unwrap)

```suggestion
        command.arg(format!("--webpack-config={}", &webpack_config_path.to_str()?.to_string()));
```

Reply:
No that does not work either `the trait `std::error::Error` is not implemented for `std::option::NoneError`

Reply:
I think it's fine the way it is, it should never be an empty path, even if you specify webpack_config="" it will default to "webpack.config.js".

Reply:
after some research you could use Option::ok_or() to convert into a Result, then use ? on that result to get what's inside.

Reply:
Why is that necessary?